manifests/golang.yml

```diff
@@ -468,6 +468,18 @@ tests/:
     test_otel_sdk_interoperability.py: missing_feature
     test_otel_span_with_baggage.py:
       Test_Otel_Span_With_Baggage: missing_feature
+    test_parametric_endpoints.py:
+      Test_Parametric_DDSpan_Add_Link: missing_feature (add_link endpoint is not implemented)
+      Test_Parametric_DDSpan_Set_Resource: missing_feature (set_resource endpoint is not implemented)
```

Comment:
Span links and resource can only be configured when starting a span. These fields can't be modified on an existing span.
So, the features aren't supported... hence why the endpoints aren't implemented